templates/static/accessibility.html

```diff
@@ -0,0 +1,138 @@
+
+  <p>{{ _("Some radio inputs on the relationships page do not conform to GDS guidelines. When selected, three different elements on the page are updated.") }}</p>
+
+  <p>{{ _("On ‘Who lives here?’ there is a ‘Previous’ link on the page that does not conform with GDS guidelines. The text used should read as ‘Back’.") }}</p>
+
+  <p>{{ _("Error messages do not result in a change to the page title to reflect that an error has been made. This does not conform to GDS guidelines.") }}</p>
+
+  <h2>{{ _("How we tested this questionnaire") }}</h2>
+
+  <p>{{ _("Over the last few years we have tested the full questionnaire, including:") }}</p>
+
+    <ul class="list">
+    <li>{{ _("<a href='{url}'>Digital Accessibility Centre (DAC)</a> carrying out a professional, technical audit").format(url = "https://digitalaccessibilitycentre.org/") }}</li>
+    <li>{{ _("user testing with the public on different devices, different locations, in their homes with various assistive technologies including free and paid versions") }}</li>
+    <li>{{ _("testing with charities, such as Scope and RNIB in their labs") }}</li>
+    <li>{{ _("using job centres and libraries for pop-up testing") }}</li>
+    <li>{{ _("covering a wide range of ages, additional needs and disabilities") }}</li>
+    <li>{{ _("making sure the offline journey and non-digital products were considered in the end-to-end journey") }}</li>
+    <li>{{ _("changing designs based on feedback from user testing, such as letter structure and the summary of sections and progress through questionnaire") }}</li>
+  </ul>
+
+  <h3>{{ _("Digital Accessibility Centre testing") }}</h3>
+
+  <p>{{ _("Parts of this questionnaire were last tested on 3 September 2019 by the Digital Accessibility Centre (DAC).") }}</p>
+
+
+  <p>{{ _("<a href='{url}' download>DAC Report - September 2019</a>").format(url = "https://00e9e64bac927483db2dfd4487850d3028707017bf2b680f4b-apidata.googleusercontent.com/download/storage/v1/b/census-ci-craftcms/o/assets%2FONS-Census-Survey-2021-DAC-Accessibility-Report-Sept-2019.pdf?qk=AD5uMEv4OOShhFARPG-naoLGLStfFwDhiJI9Ppl2oGD8z5UMt3RTdkZOHQupf4-x6E2MOcwbrnCGFPoWF1WTZp0TdnA3QVU1KYuYt3_hpMKS3zcsd1uQESRvF4qU5cRkq34VMSVMKD4ZcPkvab26rSnAt_8qNykO-1aCuOJxNzf0UxhF19b-Pvpy0B5LnzswSkgQN-OmJS7JNKzQMaBelwaSj44Xkg0WSsK32gPxSpxHhL5Bu8DE2ntc8Dm9ppaI01S3WF8xkrOUmj6dtZAXl2AvCHUc64QOddRJKJDh1CHSSMXH_o7YeccP4JVkHXZx4GpO8an62GZVDsAZaZJfesam7A9II5C8nAGmJyYREzhMjgJtz0zxITFLXOVBdj6g9jTXD8VxfdM00YWH_x7fDVPwNUKcVdCTSXWPAlmX66dyj_BEz23guQqOK5pyrPlVMfZfBlBBTyg_fk-YiFEO_Ridw6v4z4ABPkiJvWSoeM_gUiNXRs1JZbCqMuVPHAL6iwMb9TTj6Tvu6-XOkLRhhqNQqLFvVU26NP042o30K9Ilk_CWP978sRy7isjuxb_EthQr64wj0TE8TKEBfTubJANRmUCBFBgQ7vgO0lrb-ySSFhXoO_o-870HeAQ7Mefoi6CWaTmH3mbZCP3P2IXz4obWSfWjF8MgB64wiYeAyCVB2Jvg2Hfv3mkP2hasvsUlLYmcFaa9NrRr-PFy_HTOyMQdQYNddyNCjoMu6XWL2_RU8jYazgmxGLvEY-KwvAjDxApWyivBT2u2f5_0-76RR-xggzv5ehNL29FowG3YrZEYbxNJjeaA1uuFWDV_vzKSyRQ-W0Qq0eBWxe6ReK2TKAPixAvUUoYvAv4LXuRGojB-v1MW4F86-w_QAXr1f75yj_FSoglrR4cI") }}</p>
```

Comment:
The link doesn't work, and this is a horrible url. We should not be publishing a url like this when we have a cdn we could serve it from.

Reply:
Yeah I know what I've done there, ill fix that